CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.33.10] - 2025-01-03
```

Comment:
Let's bump this version to 3.34.0. This change could be considered large enough to warrant a minor bump. More importantly, this will give us room to bump the patch version for the release included in CSM 1.6.1 if critical bug fixes are needed.